canonical-websites/www.ubuntu.com
# 2905 remove snappy page

## Done

[List of work items including drive-bys]

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/snappy>
- Check that page is redirected to `/desktop/developers`


## Issue / Card

Fixes #2905
